.github/settings.yml

```diff
@@ -165,6 +165,8 @@ labels:
     color: "#ffffff"
   - name: Status:Tested
     color: "#62db68"
+  - name: Status:Staled
```

Comment:
```suggestion
  - name: Status:Stale
```

IMO "Staled" is not a real word.

Reply:
@phil-davis uhm, confusion:

![image](https://user-images.githubusercontent.com/6905948/104590031-09276c00-566b-11eb-9e00-7f361bf77089.png)

I thought about using a passive voice

Reply:
At least not in modern English

Reply:
If it sounds funny for a native, then I'd go with your gut feeling! thanks!